src/UserFrontend.php

```diff
@@ -41,6 +41,9 @@ public static function init() {
     if (!Admin::currentUserHasAccess()) {
       add_filter('show_admin_bar', '__return_false');
     }
+
+    // Moves WP admin bar to bottom of page.
+    add_action('admin_bar_init', __CLASS__ . '::move_admin_bar_to_bottom');
```

Comment:
We are using camel case for custom methods name.

Reply:
Also, why is this into `UserFrontend` and not in `Admin` class?

Reply:
All other function names in this file (and project) are using snake case?

The `Admin` class only gets loaded in the admin back-end, not on the front-end (even if the user is an admin). I already tried it.

Reply:
> All other function names in this file (and project) are using snake case?

Then I do not know why we are not following our standard here. 🤷‍♂ 
Does not matter, just ignore my comment.

> The Admin class only gets loaded in the admin back-end, not on the front-end (even if the user is an admin). I already tried it.

Ah, right. The `Admin` class is hooked on `admin_init`. You're right. 👍